tools/homework5.sh

```diff
@@ -0,0 +1,6 @@
+for i in material rxjs libimobiledevice ansible
+do
+cd ~/workspace/$i
+git checkout master
+git pull origin master
+done
```

Comment:
what is the output of this script? I don't think it can work as before you refactor.

Reply:
# input below command in fetch-repos.sh file.
$cd ~/workspace
$cd material
$git checkout master
$git pull origin master

$cd ..    # back to workspace directory
$cd rxjs  # go to rxjs directory
$git checkout master
$git pull origin master

$cd ../libimobiledevice  # go to libimobiledevice directory
$git checkout master
$git pull origin master

$cd ../ansible  # go to libimobiledevice directory
$git checkout master
$git pull origin master
######################################
I executed the above script with the for statement.